packages/zosfiles/src/api/methods/list/List.ts

```diff
@@ -118,6 +119,48 @@ export class List {
         }
     }
 
+    /**
+     * Retrieve a list of all files from a path name
+     *
+     * @param {AbstractSession}  session      - z/OS MF connection info
+     * @param {string}           path         - contains the uss path name
+     * @param {IUSSListOptions} [options={}] - contains the options to be sent
+     *
+     * @returns {Promise<IZosFilesResponse>} A response indicating the outcome of the API
+     *
+     * @throws {ImperativeError} data set name must be set
+     * @throws {Error} When the {@link ZosmfRestClient} throws an error
+     */
+    public static async fileList(session: AbstractSession, path: string, options: IUSSListOptions = {}): Promise<IZosFilesResponse> {
+        ImperativeExpect.toNotBeNullOrUndefined(path, ZosFilesMessages.missingUSSFileName.message);
+        ImperativeExpect.toNotBeEqual(path, "", ZosFilesMessages.missingUSSFileName.message);
+
+        try {
+            const endpoint = posix.join(ZosFilesConstants.RESOURCE,
+                `${ZosFilesConstants.RES_USS_FILES}?${ZosFilesConstants.RES_PATH}=${path}`);
```

Comment:
```suggestion
                `${ZosFilesConstants.RES_USS_FILES}?${ZosFilesConstants.RES_PATH}=${encodeURIComponent(path)}`);
```
same issue as https://github.com/zowe/zowe-cli/pull/245 found by @ChrisBoehmCA 
encode USS path when appending it to the resource so that paths with special characters can still be uploaded/ downloaded